### PR TITLE
Use packed stdext

### PR DESCRIPTION
**Pull request by jonludlam**
Do not merge yet. Here for review only.

### CODE REVIEW

**Comment by simonjbeaumont** [comment]
@xen-git retest this please.

**Comment by simonjbeaumont** [comment]
Adding label since it's been asked not to be merged

File: networkd/network_server.ml [hunk 1]
```diff
@@ -15,9 +15,9 @@
 open Network_utils
 open Network_interface
 
-open Fun
-open Xstringext
-open Listext
+(*open Fun*)
```
Comment (johnelse) [inline networkd/network_server.ml]: Why not remove this? I don't think `|>` is going away :-)

Comment (jonludlam) [inline networkd/network_server.ml]: Ah, I think I commented it out to see if it was using any of the other funky things declared in Fun, but seemingly not :-)

File: networkd/network_monitor.ml [hunk 2]
```diff
@@ -15,13 +15,9 @@
...
-open Xstringext
-open Threadext
-
 let write_stats stats =
 	let payload = stats |> rpc_of_stats_t |> Jsonrpc.to_string in
 	let checksum = payload |> Digest.string |> Digest.to_hex in
 	let length = String.length payload in
 	let data = Printf.sprintf "%s%s%08x%s" magic checksum length payload in
-	Unixext.write_string_to_file stats_file (data)
+	Stdext.Unixext.write_string_to_file stats_file (data)
```
Comment (lindig) [inline networkd/network_monitor.ml]: What are the `()` doing around `data`?

File: test/network_test_lacp_properties.ml [hunk 3]
```diff
@@ -91,7 +90,7 @@ let test_lacp_aggregation_key_vsctl arg () = skip_if true "Unimplemented" ;
 	print_endline answer ;
 	assert_bool "lacp_aggregation_key is passed to ovs-vsctl command"
 		(List.exists
-			(fun s -> (Xstringext.String.(strip isspace s) == answer))
+			(fun s -> (Stdext.Xstringext.String.(strip isspace s) == answer))
```
Comment (lindig) [inline test/network_test_lacp_properties.ml]: `==` tests for physical equality but why?

**Pull request merged**